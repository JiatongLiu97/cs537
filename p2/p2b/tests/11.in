/bin/echo making a new file > tests-out/tmp.txt
/bin/cat tests-out/tmp.txt
exit
