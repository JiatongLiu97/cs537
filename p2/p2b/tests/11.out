/bin/echo making a new file > tests-out/tmp.txt
/bin/cat tests-out/tmp.txt
making a new file
exit
